packages/trader/src/Modules/MT5/Components/mt5-poa.jsx

```diff
@@ -427,7 +424,11 @@ class MT5POA extends React.Component {
                                             <FormSubmitButton
                                                 has_cancel
                                                 cancel_label={localize('Previous')}
-                                                is_disabled={this.isFormDisabled(dirty, errors)}
+                                                is_disabled={
+                                                    this.isFormDisabled(dirty, errors) ||
+                                                    (this.state.document_file && this.state.document_file.length < 1) ||
```

Comment:
As we're setting `[]` for the **document_file** initial state, do we need to check for `this.state.document_file` again? Is something else changing it?

Reply:
It's implemented similarly here: https://github.com/binary-com/deriv-app/blob/master/packages/account/src/Sections/Verification/ProofOfAddress/proof-of-address-form.jsx#L451